straxen/corrections_services.py

```diff
@@ -75,8 +77,8 @@ def get_corrections_config(self, run_id, config_model=None):
 
         if 'to_pe_model' in model_type:
             return self.get_pmt_gains(run_id, model_type, global_version)
-        elif 'elife' in model_type:
-            return self.get_elife(run_id, model_type, global_version)
+        elif model_type in single_value_corrections:
+            return self._get_correction(run_id, model_type, global_version)
         elif model_type in corrections_w_file:
             return self.get_config_from_cmt(run_id, model_type, global_version)
         else:
```

Comment:
Not part of this PR, but since it just strikes my eye. Can you also add to the raise "... following values are supproted: ...." This often helps to spot the mistake.

Reply:
done